web/docs/integrations/suricata/README.md

```diff
@@ -5,7 +5,9 @@ to detect threats.
 
 Use Tenzir to acquire, process, and store Suricata logs.
 
-## Ingest EVE JSON logs into a node
+## Examples
+
+### Ingest EVE JSON logs into a node
```

Comment:
Let's not recommend `import` in an example. Can we use `publish "suricata"` instead?